target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -362,6 +362,16 @@ define Device/tplink_cpe610-v2
 endef
 TARGET_DEVICES += tplink_cpe610-v2
 
+define Device/tplink_cpe710-v1
+  $(Device/tplink-safeloader-uimage)
+  SOC := qca9563
+  IMAGE_SIZE := 15680k
+  DEVICE_MODEL := CPE710
+  DEVICE_VARIANT := v1
+  TPLINK_BOARD_ID := CPE710V1
```

Comment:
Would we need `DEVICE_PACKAGES` for ath10k here?

Reply:
It works without it but I have added it back to be consistent with the other devices.